docs/contributing/pull-requests/index.md

```diff
@@ -0,0 +1,152 @@
+# Pull Requests
+
+Pull Requests are the primary mechanism we use to write software. GitHub has some great
+[documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+on using the Pull Request feature.
+
+<community>
+
+## Fork
+
+In order to contribute to Bitwarden you will need to fork the relevant repository. For details on
+how to do this see this
+[help article from GitHub](https://docs.github.com/en/get-started/quickstart/fork-a-repo). After
+forking the repository you will need to clone it locally.
+
+```bash
+# Example for the clients repository
+git clone git@github.com:username/clients.git
+```
+
+It's also useful to add a `upstream` remote pointing to the official Bitwarden repository.
+
+```bash
+# Example for the clients repository, from the repository directory
+git remote add upstream https://github.com/bitwarden/clients.git
+```
+
+This will allow you to pull in upstream changes easily by running.
+
+```bash
+# Example for the clients repository, from the repository directory
+git fetch upstream
+```
+
+</community>
+
+## Branch
+
+Each new feature or bug fix should be developed on a separate branch. Branches allows you to work on
+multiple features concurrently. In most cases you should branch from `master`. However, if you are
+working with other contributors we typically branch of a long-lived feature branch. Long-lived
+feature branches allows us to break up a single feature into multiple PRs, which can be reviewed
+individually.
+
+<community>
+
+As a community contributor you can use the following command to branch directly from the _upstream_
+master branch.
+
+```bash
+git checkout -b feature/example
+```
+
+</community>
+
+<bitwarden>
+
+As a bitwarden contributor you should branch of `origin/master`, this ensures that the branch is
```

Comment:
```suggestion
As a Bitwarden developer you should branch of `origin/master`, this ensures that the branch is
```

Reply:
I've tried to use the terms community contributor and bitwarden contributor instead of developer, since the docs are called contributor docs. We can have a further discussion but we should try to standardize on a common phrasing.

Reply:
Makes sense!  Thank you.